site/server/app/com/fortysevendeg/exercises/controllers/ApplicationController.scala

```diff
@@ -1,29 +1,34 @@
 package com.fortysevendeg.exercises.controllers
 
+import scala.language.postfixOps
```

Comment:
We should add all these flags in the build.sbt as compiler flags. We will use infix notation and higher kinds elsewhere and there is no need to have these unnecesary advanced lang imports everywhere